content/docs/reference/Releases/regolith-3.2-release-notes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "3.2 Release Notes"
+linkTitle: "3.2 Release Notes"
+weight: 1
+description: >
+  Release notes for Regolith 3.2
+---
+
+# Regolith 3.2 Release Notes
+
+Regolith 3.2 is a minor release focusing on Ubuntu 24.04 support and bug fixes.
+
+## Features
+
+* Ubuntu 24.04 (Noble) support
+* `regolith-control-center` rebased from GNOME 46 (for supported disto releases)
+* Sway session promoted from `beta` to `stable` status
+* Overhaul of X11 and Wayland session initialization
+
+## Installation Instructions
+
+Follow the standard installation instructions, but substitute `testing` instead of `release-3_1`.  For example, this is the `apt` line for Ubuntu Mantic:
```

Comment:
Is this a typo? `Ubuntu Mantic`?

Reply:
thx, yep I just missed it in search/replace